backend/src/app.ts

```diff
@@ -102,3 +112,15 @@ function TournamentRegistrationsRepository() {
   }
   return new InMemoryTournamentRegistrationsRepository();
 }
+
+function PlayerProfilesRepository() {
+  //TODO add later above repositories ???
+  // if (process.env.MONGO_REPOSITORIES === 'ENABLED' && process.env.TOURNAMENTS_REGISTRATIONS_DATABASE === 'MONGO') {
+  //   return new MongoPlayerProfilesRepository();
+  // }
+  // if (process.env.POSTGRES_REPOSITORIES === 'ENABLED' && process.env.TOURNAMENTS_REGISTRATIONS_DATABASE === 'POSTGRES') {
+  //   return new PostgreSqlPlayerProfilesRepository();
+  // }
```

Comment:
@nowakprojects osobnego brancha na to, czy już w tym dorzeźbić :-P ?

Reply:
Nie no, nie musisz zrobić Postgresa, jak wybrałeś mongo to jest mongo :) To tak na przykład zrobiłem dwa, że możecie co chcecie wybrać. Z taką architekturą nie ma znaczenia jakiej bazy używasz. Ale jedna wystarczy :)